examples/client/client.c

```diff
@@ -1099,455 +1099,674 @@ static int ClientRead(WOLFSSL* ssl, char* reply, int replyLen, int mustRead,
+                                   "            0: 英語、 1: 日本語\n", 
+    },
 #if !defined(NO_DH) && !defined(HAVE_FIPS) && \
     !defined(HAVE_SELFTEST) && !defined(WOLFSSL_OLD_PRIME_CHECK)
-        "-2          Disable DH Prime check\n",                         /* 61 */
+        {
+            OPTION_TYPE_DISABLE_DHKEY_CHECK, '2',
+            "-2          Disable DH Prime check\n",
+            "-2          DHプライム番号チェックを無効にする\n", 
+    },
 #endif
 #ifdef HAVE_SECURE_RENEGOTIATION
-        "-4          Use resumption for renegotiation\n",               /* 62 */
+        {
+            OPTION_TYPE_USE_RESUMPTION, '4',
+            "-4          Use resumption for renegotiation\n"
+            "-4          再交渉に再開を使用\n",
+    },
 #endif
 #ifdef HAVE_TRUSTED_CA
-        "-5          Use Trusted CA Key Indication\n",                  /* 63 */
+        {
+            OPTION_TYPE_TRUSTED_CA_KEY_INDICATION, '5',
+            "-5          Use Trusted CA Key Indication\n",
+            "-5          信頼できる認証局の鍵表示を使用する\n", 
+    },
 #endif
-        "-6          Simulate WANT_WRITE errors on every other IO send\n",
+        {
+            OPTION_TYPE_SIMULATE_WANT_WRITE, '6',
+            "-6          Simulate WANT_WRITE errors on every other IO send\n",
+            "-6          WANT_WRITE エラーを全てのIO 送信でシュミレートします\n",
+    },
 #ifdef HAVE_CURVE448
-        "-8          Use X448 for key exchange\n",                      /* 66 */
+        {
+            OPTION_TYPE_USE_X448, '8',
+            "-8          Use X448 for key exchange\n",
+            "-8          鍵交換に X448 を使用する\n", 
+    },
 #endif
 #if defined(OPENSSL_ALL) && defined(WOLFSSL_CERT_GEN) && \
     (defined(WOLFSSL_CERT_REQ) || defined(WOLFSSL_CERT_EXT)) && \
     !defined(NO_FILESYSTEM) && !defined(NO_WOLFSSL_DIR)
-        "-9          Use hash dir look up for certificate loading\n"
-        "            loading from <wolfSSL home>/certs folder\n"
-        "            files in the folder would have the form \"hash.N\" file name\n"
-        "            e.g symbolic link to the file at certs folder\n"
-        "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
-                                                                        /* 67 */
+        {
+            OPTION_TYPE_USE_HASH_DIR, '9',
+            "-9          Use hash dir look up for certificate loading\n"
+            "            loading from <wolfSSL home>/certs folder\n"
+            "            files in the folder would have the form \"hash.N\" file name\n"
+            "            e.g symbolic link to the file at certs folder\n"
+            "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
+            "-9          証明書の読み込みに hash dir 機能を使用する\n"
+            "            <wolfSSL home>/certs フォルダーからロードします\n"
+            "            フォルダー中のファイルは、\"hash.N\"[N:0-9]名である必要があります\n"
+            "            以下の例ではca-cert.pemにシンボリックリンクを設定します\n"
```

Comment:
Please keep the line under our 80-character line length limit per our Coding Standard